src/Constants.ts

```diff
@@ -77,3 +77,20 @@ export const HARDCODED_RULES = {
 		category: 'Scanner Internal'
 	}
 };
+
+export enum Severity {
+	NONE = 0,
+	LOW = 3,
+	MODERATE = 2,
+	HIGH = 1
+}
+
+export const severityMap = new Map([
```

Comment:
Nitpick: I feel like this should be `SeverityMap` to maintain consistency with the other constants' casing.